docs/migration.html

```diff
@@ -1,34 +0,0 @@
-<!--
```

Comment:
As far as I can tell this page is not accessible on the website. Anyway it's about Kafka 0.8!